File: 程式語言 V2答案檔/Python/109.py
n = int(input())

if n < 60:
    print('fail')
elif n <= 100:
    print('pass')
else:
    print('error')

if n <= 100 and n % 2 == 0:
    print('even')
elif n <= 100 and n % 2 != 0:
    print('odd')

File: 程式語言 V2答案檔/C#/109.cs
using System;

public class TQC109
{
    public static void Main(string[] args)
    {
        int score;
        score = int.Parse(Console.ReadLine());
        if (score < 0 || score > 100) {
            Console.WriteLine("error");
        }
        else {
            // print score is pass or fail
            if (60 <= score && score <= 100) {
                Console.WriteLine("pass");
            }
            else {
                Console.WriteLine("fail");
            }

            // print score is even or odd 
            if (score % 2 == 0) {
                Console.Write("even");
            }
            else {
                Console.Write("odd");
            }
        }
    }
}